provider/ecs/ecs.go

```diff
@@ -291,6 +291,7 @@ func (e *ECS) CreateTaskDefinition(d service.Deploy) (*string, error) {
 		containerDefinition := &ecs.ContainerDefinition{
 			Name:  aws.String(container.ContainerName),
 			Image: aws.String(imageUri),
+			DockerLabels: container.DockerLabels,
```

Comment:
I'd use aws.StringMap(container.DockerLabels), then it's similar to the other types used (values instead of pointers)